lib/excon/middlewares/redirect_follower.rb

```diff
@@ -17,7 +17,11 @@ def response_call(datum)
             params = datum.dup
             params.delete(:stack)
             params.delete(:connection)
-            params[:method] = :get if [301, 302, 303].include? response[:status]
+            if [301, 302, 303].include?(response[:status])
```

Comment:
I'd be up for changing it. I think 301/302 is technically more correct and I'm not sure I've ever even seen a 303 in the wild.